packages/abi-gen/src/index.ts

```diff
@@ -71,11 +79,42 @@ function registerPartials(): void {
     }
 }
 
-Handlebars.registerHelper('parameterType', utils.solTypeToTsType.bind(utils, ParamKind.Input, args.backend));
-Handlebars.registerHelper('assertionType', utils.solTypeToAssertion.bind(utils));
-Handlebars.registerHelper('returnType', utils.solTypeToTsType.bind(utils, ParamKind.Output, args.backend));
+if (args.language === 'TypeScript') {
+    Handlebars.registerHelper('parameterType', utils.solTypeToTsType.bind(utils, ParamKind.Input, args.backend));
+    Handlebars.registerHelper('assertionType', utils.solTypeToAssertion.bind(utils));
+    Handlebars.registerHelper('returnType', utils.solTypeToTsType.bind(utils, ParamKind.Output, args.backend));
+} else if (args.language === 'Python') {
```

Comment:
Not necessary because `yargs` handle that for us.  I passed `--language whatever` and got:
```
Invalid values:
  Argument: language, Given: "whatever", Choices: "TypeScript", "Python"
```